src/ucp/api/ucp.h

```diff
@@ -2885,36 +2900,53 @@ ucs_status_t ucp_mem_advise(ucp_context_h context, ucp_mem_h memh,
  * with the memory handle the application is responsible for sharing the RKEY with
  * the peers that will initiate the access.
  *
- * @param [in]  context       Application @ref ucp_context_h "context" which was
- *                            used to allocate/map the memory.
  * @param [in]  memh          @ref ucp_mem_h "Handle" to memory region.
- * @param [out] rkey_buffer_p Memory buffer allocated by the library.
- *                            The buffer contains packed RKEY.
- * @param [out] size_p        Size (in bytes) of the packed RKEY.
+ * @param [in]  params        Memory handle packing parameters, as defined by
+ *                            @ref ucp_memh_pack_params_t.
+ * @param [out] buffer_p      Memory buffer allocated by the library.
+ *                            The buffer contains the packed memory handle.
+ * @param [out] buffer_size_p Size (in bytes) of the buffer which contains
+ *                            packed memory handle.
  *
  * @return Error code as defined by @ref ucs_status_t
  */
-ucs_status_t ucp_rkey_pack(ucp_context_h context, ucp_mem_h memh,
-                           void **rkey_buffer_p, size_t *size_p);
+ucs_status_t ucp_memh_pack(ucp_mem_h memh, ucp_memh_pack_params_t *params,
+                           void **buffer_p, size_t *buffer_size_p);
 
 
 /**
  * @ingroup UCP_MEM
- * @brief Release packed remote key buffer.
+ * @brief Memory handle pack parameters passed to @ref ucp_memh_pack.
```

Comment:
Please add full description in addition for brief. On the UCP level we don't accept just brief doxygen description.